src/test/java/util/xcmailr/util/SendEmail.java

```diff
@@ -0,0 +1,78 @@
+package util.xcmailr.util;
+
+import java.util.Properties;
+
+import javax.mail.Address;
+import javax.mail.BodyPart;
+import javax.mail.Message;
+import javax.mail.MessagingException;
+import javax.mail.Multipart;
+import javax.mail.Session;
+import javax.mail.Transport;
+import javax.mail.internet.InternetAddress;
+import javax.mail.internet.MimeBodyPart;
+import javax.mail.internet.MimeMessage;
+import javax.mail.internet.MimeMultipart;
+
+import util.xcmailr.data.EmailAccount;
+import util.xcmailr.data.SmtpAuthenticator;
+
+public class SendEmail
+{
+
+    /**
+     * Method to send message
```

Comment:
1. Stick with the guidelines of Oracle's [style guide](https://www.oracle.com/technical-resources/articles/java/javadoc-tool.html#styleguide)
 > Method descriptions begin with a verb phrase
2. Be more specific. We send an e-mail, not just any message.

```suggestion
     * Sends an e-mail from the specified <code>emailAccount</code> to the specified <code>recipient</code>.
```